src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -70,8 +72,8 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     private cdRef: ChangeDetectorRef,
     private pageValidationService: PageValidationService,
     private dialog: MatDialog,
-  ) {
-  }
+    private caseFieldService: CaseFieldService
```

Comment:
Not sure if this was in the PR when I reviewed previously but this should be `private readonly`, as should all of the parameters for this constructor.

Reply:
updated